content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -153,109 +153,153 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
+  ```
+  where `-ConfigFile` points to the path of the Kubernetes configuration file.
 
-    Tip: You can use tools such as [xcopy](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/xcopy), [WinSCP](https://winscp.net/eng/download.php), or this [PowerShell wrapper for WinSCP](https://www.powershellgallery.com/packages/WinSCP/5.13.2.0) to transfer the config file between nodes.
+  ![alt_text](../kubecluster.ps1-install.gif "KubeCluster.ps1 install output")
 
-5. Download Kubernetes binaries
 
-    To be able to run Kubernetes, you first need to download the `kubelet` and `kube-proxy` binaries. You download these from the Node Binaries links in the CHANGELOG.md file of the [latest releases](https://github.com/kubernetes/kubernetes/releases/). For example 'kubernetes-node-windows-amd64.tar.gz'. You may also optionally download `kubectl` to run on Windows which you can find under Client Binaries.
+This step will perform the following actions on a targeted Windows node:
+  * Enable Windows Server containers role (and reboot)
+  * Download and install the chosen container runtime
+  * Download Kubernetes binaries and add them to the $PATH environment variable
+  * Download CNI plugins based on the selection made in the Kubernetes Configuration file
+  * (Optionally) Generate a new SSH key which is required to connect to the control-plane ("Master") node during joining
+      {{< note >}}
+      For the final step where a new ssh key is generated, the public key also needs to be added to the `authorized_keys` list file on the Linux control-plane node. This action only needs to be performed once. Steps for this are printed by the script itself at the end.  
+      {{< /note >}}
 
-    Use the [Expand-Archive](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/expand-archive?view=powershell-6) PowerShell command to extract the archive and place the binaries into `C:\k`.
 
-#### Join the Windows node to the Flannel cluster
+#### Join the Windows Node to the Kubernetes cluster
+In this section we'll cover how to join a [Windows Node with Kubernetes installed](#preparing-a-windows-node) with an existing Linux control-plane ("Master") node to form a cluster.
 
-The Flannel overlay deployment scripts and documentation are available in [this repository](https://github.com/Microsoft/SDN/tree/master/Kubernetes/flannel/overlay). The following steps are a simple walkthrough of the more comprehensive instructions available there.
+Use the previously downloaded [KubeCluster.ps1](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/KubeCluster.ps1) script to join the Windows node to the cluster:
 
-Download the [Flannel start.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/start.ps1) script, the contents of which should be extracted to `C:\k`:
+    ```PowerShell
+    .\KubeCluster.ps1 -ConfigFile .\Kubeclustervxlan.json -join 
```

Comment:
does this doc mention the current state of vxlan vs overlay and the fact that vxlan requires a newer version of Windows?

Reply:
should we recommend overlay as the default?

Reply:
Hmmm... My thought was that l2bridge needs additional work creating UDRs on most public cloud platforms, overlay should work anywhere out of the box. We should show simplest path to getting Kubernetes working as default. The previous docs also showed overlay as default. What are your thoughts for l2bridge as default?

But yes, I will show that overlay requires v1.14 at least and WS2019!

Reply:
yeah, i think we should outline some of the trade-offs we have for the methods, currently.

Reply:
Sure, this item should be in https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#networking though or is something missing?

Reply:
the table in there is good. i forgot about it.
ideally it should have a minimal supported Windows Version column too.

Reply:
Thanks for the suggestion! But in https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-containers-in-kubernetes it states that:
"Windows Server 2019 is the only Windows operating system supported"

So since we require Windows Server 2019 for Kubernetes overall irrespective of networking choice, wouldn't that make the column superfluous?

Reply:
I added a note under the deployment instructions directly as well to highlight this though:
https://github.com/kubernetes/website/pull/15750/commits/e3d9cb40e1bdf9c39f9474eda7e68a9ee1e666f6#diff-f8c172f80db498dfcda26f3813cfa729R252

Reply:
there are specific versions of Windows Server 2019 that does not support vxlan.
i encountered this issue when testing kubeadm worker nodes.

Reply:
OK, in this case, I will add a note in the table again that you need KB4489899 installed. The note in the deployment states this already as well, thanks!

Reply:
added in https://github.com/kubernetes/website/pull/15750/files#diff-b563bff111c0a208abb2db63f1e2bad4R136